<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>481</width>
      <height>861</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>运行结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>680</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>载入代码(LOAD)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cmd">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>730</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>命令输入窗口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>680</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>执行代码(RUN)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cmd">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>770</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_sentence">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>语句与语法树</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_terminal">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>860</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>680</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清空代码(CLEAR)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>代码</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_sentence">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_terminal">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>830</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>终端</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_state">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>40</y>
      <width>291</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>运行状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_debug">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>680</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>调试/单步(Debug/Step)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
